docs/introduction/how-tos/how_to_contribute.md

```diff
@@ -1,6 +1,21 @@
 How to Contribute
 =================
 
+- [1. How to contribute to the siembol Java project](#how-to-contribute-to-the-siembol-java-project)
+    * [1.1 Environment](#environment)
+    * [1.2 How to compile and install](#how-to-compile-and-install)
```

Comment:
Can we skip How to ? eg Compiling and installing

Reply:
Done!